manifests/client.pp

```diff
@@ -157,18 +157,18 @@
     class { '::nagios::check::ram': }
     class { '::nagios::check::swap': }
     # Conditional ones, once presence is detected using our custom facts
-    if $::nagios_couchbase {        class { '::nagios::check::couchbase': } }
-    if $::nagios_pci_hpsa {         class { '::nagios::check::hpsa': } }
-    if $::nagios_httpd {            class { '::nagios::check::httpd': } }
-    if $::nagios_pci_megaraid_sas { class { '::nagios::check::megaraid_sas': } }
-    if $::nagios_memcached {        class { '::nagios::check::memcached': } }
-    if $::nagios_mongod {           class { '::nagios::check::mongodb': } }
-    if $::nagios_mountpoints {      class { '::nagios::check::mountpoints': } }
-    if $::nagios_moxi {             class { '::nagios::check::moxi': } }
-    if $::nagios_httpd_nginx {      class { '::nagios::check::nginx': } }
-    if $::nagios_pci_mptsas {       class { '::nagios::check::mptsas': } }
-    if $::nagios_mysqld {
-      case $::operatingsystem {
+    if $facts['nagios_couchbase'] {        class { '::nagios::check::couchbase': } }
```

Comment:
class doesn't need :: either
in fact, class notation here is unnecessary, should be ```include nagios::check::couchbase``` instead